happi/cli.py

```diff
@@ -1134,7 +1137,12 @@ def repair(
             break
 
 
-@happi_cli.command()
+@click.group(help="Commands related to the happi config file.")
```

Comment:
`happi -h` only shows `config` as a separate command, hiding the subcommands.  `happi config -h` shows those subcommands.  I think this is probably ok, as we're not exactly advertising these to the average user.  

If we wanted to be more transparent, we could list the subcommand names in the help message, but I don't think that's strictly necessary

Reply:
I agree that it's good enough, but will change if there's somebody with a strong desire.